### PR TITLE
Minor fixes to build and packaging

### PR DESCRIPTION
**Pull request by martinsumner**
See https://github.com/nhs-riak/riak/issues/1

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -242,6 +242,7 @@ PKG_ID := "$(REPO_TAG)-OTP$(OTP_VER)"
 PKG_VERSION = $(shell echo $(PKG_ID) | sed -e 's/^$(REPO)-//')
 
 package:
+	compile
```
Comment (stephenwood24-nhs) [inline Makefile]: I don't think this `compile` is neccessary. I've built debian package dozens of times with `make package` and had no issues

**Review by stephenwood24-nhs — APPROVED** [review]

**Comment by martinsumner** [comment]
The change to the break command doesn't work as expected, in that we're changing the type of a previous configuration parameter - so anyone copying over a previous riak.conf will get a `Cuttlefish failed!` message if they attempt to start

File: priv/riak.schema [hunk 2]
```diff
@@ -118,11 +118,23 @@
...
   merge
 ]}.
 
+{translation,
+  "vm_args.+B",
+  fun(Conf) ->
+    Setting = cuttlefish:conf_get("erlang.vm.ignore_break_signal", Conf),
+    case Setting of
+      true -> "i";
+      false -> "c"
```
Comment (tburghart) [inline priv/riak.schema]: I think `false` should call `cuttlefish:unset()`

Comment (martinsumner) [reply priv/riak.schema]: Changed as suggested, looking at `ps` output after starting riak:

Setting to `true` -

```riak -scl false -sfwi 500 -P 256000 -e 256000 -Q 262144 -A 64 -K true -W w -B i -zdbbl 32768 ...```

Setting to `false` -

```riak -scl false -sfwi 500 -P 256000 -e 256000 -Q 262144 -A 64 -K true -W w -zdbbl 32768 ...```

**Pull request merged**